### PR TITLE
fix storage list code snippet

### PR DESCRIPTION
**Pull request by elorzafe**
#### Description of changes:
fix storage `list` method code snippet
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by ashwinkumar6 — APPROVED** [review]

**Review by kvramyasri7 — APPROVED** [review]
LGTM

File: src/pages/[platform]/build-a-backend/storage/list/index.mdx [hunk 1]
```diff
@@ -129,7 +129,7 @@ To list other user's objects:
 import { list } from 'aws-amplify/storage';
 
 try {
-  const response = list({ prefix: 'photos/', options:  {
+  const response = await list({ prefix: 'photos/', options:  {
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/storage/list/index.mdx:132]: on this page, lots of the code formatting is also slightly off. Do you mind fixing all those as well as you're at it?

Comment (renebrandel) [reply src/pages/[platform]/build-a-backend/storage/list/index.mdx:132]: for example `prefix` on a new line etc.

Comment (elorzafe) [reply src/pages/[platform]/build-a-backend/storage/list/index.mdx:132]: sounds good

**Review by ashwinkumar6 — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**